draft-ietf-mls-architecture.md

```diff
@@ -765,6 +765,34 @@ proposals, but any other policies must be assured to be consistent as above.
 > Have an explicit group policy setting the conditions under which external
 > joins are allowed.
 
+## Handling Authentication Failures
+
+Within an MLS group, every member is authenticated to other member, by means of
+credentials issued and verified by the Authentication Service.  MLS does not
+prescribe what actions, if any, an application should take in the event that a
+group member presents an invalid credential.  For example, an application may
+require such a member to be immediately evicted, or may allow some grace period
+for the problem to be remediated. To avoid operational problems, it is important
```

Comment:
Is this supposed to be subsequent to group establishment? I would assume that at formation time, invalid users would simply be rejected.

Reply:
This is for external joins